### PR TITLE
Migrate JPA Inheritance FAT to Open Liberty

### PR DESCRIPTION
**Pull request by jgrassel**
Migrate JPA Inheritance JPA10 bucket to Open Liberty.

Signed-off-by: Joe Grassel <jgrassel@us.ibm.com>

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
JPA FAT in build is good.

File: dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/jpa10/Inheritance_EJB.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/jpa10/Inheritance_EJB.java]: 2019

File: dev/com.ibm.ws.jpa_22_fat/test-applications/jpa10/inheritance/web/inheritance.war/WEB-INF/classes/META-INF/persistence.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_22_fat/test-applications/jpa10/inheritance/web/inheritance.war/WEB-INF/classes/META-INF/persistence.xml]: Done

File: dev/com.ibm.ws.jpa_22_fat/test-applications/jpa10/inheritance/web/inheritance.war/WEB-INF/classes/META-INF/orm.xml [hunk 3]
```diff
@@ -0,0 +1,318 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<entity-mappings
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_22_fat/test-applications/jpa10/inheritance/web/inheritance.war/WEB-INF/classes/META-INF/orm.xml]: Done

File: dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/jpa10/Inheritance_EJB.java (hunk 1 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/jpa10/Inheritance_EJB.java]: Done

**Pull request merged**